src/__tests__/Tooltip.spec.js

```diff
@@ -236,6 +236,18 @@ describe('Tooltip', () => {
     wrapper.unmount();
   });
 
+  it('should pass down offset', ()=> {
+    const wrapper = mount(
+      <Tooltip
+        isOpen
+        target="target"
+        offset='100'>Tooltip content</Tooltip>
```

Comment:
- Unexpected usage of singlequote , use " " instead
- The closing bracket must be aligned with the line containing the opening tag